frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -646,6 +658,31 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
                   }
                 />
               </StackItem>
+              <StackItem>
+                <Checkbox
+                  id="cnv668"
+                  description={
+                    <>
+                      {t('kubevirt-plugin~Refer to the')}{' '}
+                      <ExternalLink
+                        text={t('kubevirt-plugin~Documentation')}
+                        href={PREALLOCATION_DATA_VOLUME_LINK}
+                      />{' '}
+                      {t(
+                        'kubevirt-plugin~or contact your system administratorfor more information.',
+                      )}
+                      <br />
+                      {t(
+                        'kubevirt-plugin~Note: Enabling preallocation is available only for blank disk source',
+                      )}
+                    </>
```

Comment:
a. You can use the `Trans` 
b. don't use line brakes, it can be one long sentance (without the need for "Note:")